pandas/io/json/_json.py

```diff
@@ -501,7 +501,7 @@ def read_json(
         By file-like object, we refer to objects with a ``read()`` method,
         such as a file handle (e.g. via builtin ``open`` function)
         or ``StringIO``.
-    orient : str
+    orient : str, default None
         Indication of expected JSON string format.
         Compatible JSON strings can be produced by ``to_json()`` with a
         corresponding orient value.
```

Comment:
``The set of possible orients is:`` This read to me as None is not an option, hence please clarify